Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.Core/Model/Search/NotificationMessageSearchCriteria.cs

```diff
@@ -0,0 +1,16 @@
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.NotificationsModule.Core.Model.Search
+{
+    public class NotificationMessageSearchCriteria : SearchCriteriaBase
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public bool IsActive { get; set; }
+        /// <summary>
+        /// time interval used to evaluate  active notifications have failure delivery
+        /// </summary>
+        public int RepeatHoursIntervalForFail { get; set; }
```

Comment:
Why we can't use Start and End date instead? Use RepeatHoursIntervalForFail  reduce the flexibility of usage and complicate the level of data access due to mixing business and data access logic?

Reply:
actually, don't need it. have 
`function getSearchCriteria()
        {
            var searchCriteria = {
                catalogId: blade.catalogId,
                categoryId: blade.categoryId,
                keyword: filter.keyword ? filter.keyword : undefined,
                sort: uiGridHelper.getSortExpression($scope),
                skip: ($scope.pageSettings.currentPage - 1) * $scope.pageSettings.itemsPerPageCount,
                take: $scope.pageSettings.itemsPerPageCount
            };
            return searchCriteria;
        }`